web/src/tippyjs.ts

```diff
@@ -184,6 +184,24 @@ export function initialize(): void {
         },
     });
 
+    tippy.delegate("body", {
+        target: ".subscription_block",
+        trigger: "mouseenter",
+        delay: LONG_HOVER_DELAY,
+        placement: "right",
+        appendTo: () => document.body,
+        onShow(instance) {
+            const stream_name = instance.reference.querySelector(".stream-name");
+            assert(stream_name instanceof HTMLElement);
+            if (stream_name.offsetWidth >= stream_name.scrollWidth) {
+                return false;
+            }
+            const truncated_stream_name = stream_name.textContent ?? "";
+            instance.setContent(truncated_stream_name);
+            return undefined;
+        },
+    });
```

Comment:
This needs to be reworked to interact properly with 482f6c5449fac1afa0acaf08d1c2e909ac617ae0; the logic should be in that existing function, rather than creating a duplicate tooltip function.